projects/plugins/protect/src/js/components/admin-page/index.jsx

```diff
@@ -213,6 +213,17 @@ const ProtectAdminPage = () => {
 	);
 };
 
+const useCredentialState = () => {
+	const { checkCredentialsState } = useDispatch( STORE_ID );
+	const credentialState = useSelect( select => select( STORE_ID ).getCredentialState() );
+
+	useEffect( () => {
+		if ( ! credentialState.state ) {
+			checkCredentialsState();
+		}
+	} );
```

Comment:
```suggestion
	} );
	}, [ checkCredentialsState, credentialState.state ] );
```
I think we should include the second argument of `useEffect()` here, to ensure it runs again when the dependencies change.